org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/internal/XMLLanguageServer.java

```diff
@@ -99,6 +99,8 @@ public XMLLanguageServer() {
 		if (logLevelString != null) {
 			commands.add("-Dlog.level=" + logLevelString); // defined in org.eclipse.lemminx.logs.LogHelper
 		}
+		commands.add("-Duser.name=" + System.getProperty("user.name"));
```

Comment:
Wouldn't it be better if the process would simply inherit *all* system properties?